src/app/teams/teams-reports.component.html

```diff
@@ -1,6 +1,8 @@
 <div class="action-buttons" *ngIf="editable">
   <button mat-raised-button color="primary" i18n (click)="openAddReportDialog()">Add Report</button>
-  <button mat-raised-button color="primary" i18n (click)="exportReports()">Export as CSV</button>
+  <button mat-raised-button color="accent" i18n *ngIf="reports && reports.length > 0" (click)="exportAsCSV()">
```

Comment:
Wondering why you change the click function into a `exportAsCSV()`, export now doesnt work, as it does nothing.

<img width="1212" alt="Screenshot 2024-10-28 at 11 51 02 AM" src="https://github.com/user-attachments/assets/148169eb-b4b5-4616-abef-953c620d54d7">

please change it back to `exportReports()`

Reply:
Also, I think this is some formatting errors that my IDE is doing. I'll go back and change these as well...

Reply:
sounds good!